src/types.ts

```diff
@@ -17,22 +17,64 @@ export type SignedMessage = {
 };
 
 export type WallyConnectorOptions = {
-  // The clientId you get when signing up
+  /**
+   * The clientId you get when signing up. Should match the dashboard value
+   */
   clientId: string;
-  // Required when using devUrl or token
+  /**
+   * Required when using devUrl.
+   * TODO: Make this better.
+   * @default false
+   */
   isDevelopment?: boolean;
-  // The local instance of the wally api. APP ROOT used if unset or not dev mode
+  /**
+   * The local instance of the wally api. APP ROOT used if unset or not dev mode
+   */
   devUrl?: string;
-  // If you're testing stuff out in dev mode and don't have access to the
-  // login flow (i.e. chrome extension), but you do have a token, put it here
-  token?: string;
-  // If you want more insight into what's going on (requests and responses)
+  /**
+   * If you have another cross-tab messaging solution for informing the primary browser
+   * tab when the secondary
+   * @default false
+   */
+  disableSharedWorker?: boolean;
+  /**
+   * If you want more insight into what's going on (requests and responses)
+   * @default false
+   */
   verbose?: boolean;
-};
-
-export type RedirectOptions = {
-  closeWindow?: boolean;
-  appendContent?: boolean;
+  /**
+   * If you'd like to use a more dynamic redirect. Only possible when the
+   * redirectUrl in your dashboard settings is set as a regex.
+   */
+  redirectURL?: string;
+  /**
+   * Set to true if you'd like the SDK to automatically redirect to the same location
+   * as the primary window. Only possible when the redirectUrl in your dashboard settings
+   * is set as a regex.
+   * @default false
+   */
+  redirectToCurrentLocation?: boolean;
```

Comment:
RegEx can also be an exact match too right? In that scenario, it's a little redundant to have `redirectToCurrentLocation` too when they can set it to the `window.location.href`.

Reply:
yeah I agree for sure - was I was thinking, was do you think there's a case though where they could initialize wally with something like the home page of the app, but then the user navigates around to another page where they login from, where they would rather have the user redirected to?

Reply:
Actually, I'm not sure there's a use case and I think Mayank and I are on slightly different sides of the spectrum. As an engineer, I think all the flexibility makes it more confusing for me to integrate the SDK but if there's a single well understood way of why it's necessary, then I can implement the said custom logic myself. The flexibility takes away any clear direction of whats the simplest and best approach.

Reply:
yeah for sure, i'll take it out. i think the tough part is that there isn't really _any_ use case yet because no one's using it and we aren't sure what sorts of flexibility will be necessary